<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>organisation1</class>
 <widget class="QWidget" name="organisation1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>19</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>organisation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 204, 255)</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Username...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>org-username...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>440</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>country...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>270</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>color...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>100</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>phonenumber...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>270</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>bio...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>440</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>name...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>link...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>81</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Password...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>710</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>440</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>100</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))</string>
   </property>
   <property name="text">
    <string>choose profile img</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
